modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFieldValuesProvider.java

```diff
@@ -204,6 +210,24 @@ private Object _getUserProfileImage(long userId) {
 		return null;
 	}
 
+	private Object _getValue(
+		ObjectField objectField, Map<String, Serializable> values) {
+
+		if (Objects.equals(
+				_getInfoFieldType(objectField), ImageInfoFieldType.INSTANCE)) {
+
+			try {
+				return JSONFactoryUtil.createJSONObject(
```

Comment:
Hey Matheus, would you mind to use WebImage to follow the same pattern existing in other classes which implements InfoItemFieldValuesProvider interface?

Reply:
Done!